web/packages/teleport/src/Console/DocumentDb/DocumentDb.test.tsx

```diff
@@ -0,0 +1,139 @@
+/**
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import { screen } from '@testing-library/react';
+
+import '@testing-library/jest-dom';
+
+import 'jest-canvas-mock';
+
+import { act, render } from 'design/utils/testing';
+
+import { ContextProvider } from 'teleport';
+import { TestLayout } from 'teleport/Console/Console.story';
+import ConsoleCtx from 'teleport/Console/consoleContext';
+import { createTeleportContext } from 'teleport/mocks/contexts';
+
+import Tty from 'teleport/lib/term/tty';
+
+import { useDbSession, Status } from './useDbSession';
+
+import { DocumentDb } from './DocumentDb';
+
+import type { Session } from 'teleport/services/session';
+
+jest.mock('./useDbSession');
+
+const mockUseDbSession = useDbSession as jest.MockedFunction<
+  typeof useDbSession
+>;
+
+describe('DocumentDb', () => {
```

Comment:
Let's remove this outer `describe`. We don't have any other tests than for `DocumentDb` here, so this additional level of nesting doesn't give us anything.